repairnator/src/main/java/fr/inria/spirals/repairnator/process/ProjectScanner.java

```diff
@@ -297,19 +298,20 @@ public String readWorkspaceFromInput(String input) throws IOException {
         return properties.getProperty("workspace");
     }
 
-    public List<Build> readBuildFromInput(String input) throws IOException {
-        List<Build> result = new ArrayList<Build>();
+    public List<BuildToBeInspected> readBuildFromInput(String input) throws IOException {
+        List<BuildToBeInspected> buildsToBeInspected = new ArrayList<BuildToBeInspected>();
 
         Properties properties = getPropertiesFromInput(input);
         String buildId = properties.getProperty("buildid");
         if (buildId != null) {
             Build build = BuildHelper.getBuildFromId(Integer.parseInt(buildId), null);
             if (build != null) {
```

Comment:
ok in fact let's change there the condition with a `&& build.getStatus() == BuildStatus.FAIL`, then you can change the scannedbuildstatus to ONLY_FAIL too. And it should be ok with the inspector. WDYT?